src/main/java/com/minecolonies/blockout/PaneParams.java

```diff
@@ -431,16 +432,16 @@ else if ((attr.startsWith("rgb(") || attr.startsWith("rgba(")) && m.find())
 
     private static int getRGBA(final String attr, final Matcher m)
     {
-        final int r = MathHelper.clamp_int(Integer.parseInt(m.group(1)), 0, 255);
-        final int g = MathHelper.clamp_int(Integer.parseInt(m.group(2)), 0, 255);
-        final int b = MathHelper.clamp_int(Integer.parseInt(m.group(3)), 0, 255);
+        final int r = MathHelper.clamp(Integer.parseInt(m.group(1)), 0, 255);
+        final int g = MathHelper.clamp(Integer.parseInt(m.group(2)), 0, 255);
+        final int b = MathHelper.clamp(Integer.parseInt(m.group(3)), 0, 255);
 
         int color = ((r & 0xFF) << 16) | ((g & 0xFF) << 8) | (b & 0xFF);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Assign this magic number 16 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Assign this magic number 8 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)